lib/pages/home_screen.dart

```diff
@@ -186,19 +181,27 @@ class _HomeScreenState extends State<HomeScreen> {
             bottom: (isCupertino(context) ? _panelController.panelHeight - 18 : _panelController.panelHeight) + 80,
             child: Consumer(
                 builder: (context, ref, child) {
-                  final followOnLocation = ref.watch(followOnLocationProvider);
+                  final followOnLocation = ref.watch(followOnLocationTargetProvider);
                   return PlatformFAB(
                     onPressed: () async {
                       final status = await ref.read(locationPermissionStatusProvider.notifier).checkPermission();
+                      final followOnLocationTarget = ref.watch(followOnLocationTargetProvider);
```

Comment:
This was already watched in scope and should not be watched again in the onPressed